scripts/quick-helm-install.sh

```diff
@@ -0,0 +1,118 @@
+set -euo pipefail
+
+SMALL_INSTALL=false
+
+while test $# -gt 0; do
+    case "$1" in
+        -h|--help)
+            echo "quick-helm-install.sh"
+            echo " "
+            echo "Installs StackRox via Helm charts, requires Helm."
+            echo " "
+            echo "options:"
+            echo "-h, --help                show brief help"
+            echo "-s, --small               installs StackRox for small clusters"
+            exit 0
+            ;;
+        -s|--small)
+            SMALL_INSTALL=true
+            break
+            ;;
+    esac
+done
+
+echo "Adding the stackrox/helm-charts/opensource repository to Helm."
+
+helm repo add stackrox https://raw.githubusercontent.com/stackrox/helm-charts/main/opensource/
+
+echo "Generating STACKROX_ADMIN_PASSWORD"
+
+STACKROX_ADMIN_PASSWORD="$(openssl rand -base64 20 | tr -d '/=+')"
+
+echo "Installing stackrox-central-services"
+
+helm install -n stackrox --create-namespace stackrox-central-services stackrox/stackrox-central-services --set central.adminPassword.value="${STACKROX_ADMIN_PASSWORD}"
+
+if $SMALL_INSTALL; then
+    echo "Upgrading stackrox to a small install"
+    helm upgrade -n stackrox stackrox-central-services stackrox/stackrox-central-services \
+      --set central.resources.requests.memory=1Gi \
+      --set central.resources.requests.cpu=1 \
+      --set central.resources.limits.memory=4Gi \
+      --set central.resources.limits.cpu=1 \
+      --set scanner.autoscaling.disable=true \
+      --set scanner.replicas=1 \
+      --set scanner.resources.requests.memory=500Mi \
+      --set scanner.resources.requests.cpu=500m \
+      --set scanner.resources.limits.memory=2500Mi \
+      --set scanner.resources.limits.cpu=2000m
+fi
+
+kubectl -n stackrox rollout status deploy/central --timeout=3m
+
+echo "Forwarding ports to central"
+
+kubectl -n stackrox port-forward deploy/central --pod-running-timeout=1m0s 8000:8443 > /dev/null 2>&1 &
+
+echo "Generating an init bundle with shared secrets"
+
+kubectl -n stackrox exec deploy/central -- roxctl --insecure-skip-tls-verify \
+  --password "${STACKROX_ADMIN_PASSWORD}" \
+  central init-bundles generate stackrox-init-bundle --output - 1> stackrox-init-bundle.yaml
+
+
+if $SMALL_INSTALL; then
+    echo "Installing the first small secured cluster"
+    helm install -n stackrox stackrox-secured-cluster-services stackrox/stackrox-secured-cluster-services \
+        -f stackrox-init-bundle.yaml \
+        --set clusterName="First-Secured-Cluster" \
+        --set sensor.resources.requests.memory=500Mi \
+        --set sensor.resources.requests.cpu=500m \
+        --set sensor.resources.limits.memory=500Mi \
+        --set sensor.resources.limits.cpu=500m
+else
+    echo "Installing the first secured cluster"
+    helm install -n stackrox stackrox-secured-cluster-services stackrox/stackrox-secured-cluster-services \
+        -f stackrox-init-bundle.yaml \
+        --set clusterName="First-Secured-Cluster"
```

Comment:
[nit] Sorry, just noticed. The cluster's name in manual procedures is different `CLUSTER_NAME="my-secured-cluster"`. Could you please make names consistent in both places? (I don't have a preference which name to use as long as it is the same in the script and in manual steps)

Reply:
Done